style.css

```diff
@@ -207,4 +207,232 @@ body {
 .container a:hover {
   background-color: bisque;
   color: black;
+}
+
+ul {
+  display: grid;
+  grid-template-columns: repeat(3, 40px);
+  gap: 0;
+  margin: 5px 0;
+  list-style: none;
+}
```

Comment:
Please make sure every content in your project should be placed with flexbox, only your main container grid-container-card enclosing all your projects should be `display: grid`